tests/conftest.py

```diff
@@ -153,29 +153,45 @@ def ceremony_inputs() -> Tuple[List[str], List[str], List[str], List[str]]:
         "2",  # Please enter root threshold
         f"{_PEMS / 'JC.pub'}",  # Please enter path to public key
         "my rsa key",  # Please enter key name
-        "0",  # Please press 0 to add key, or remove key by entering its index  # noqa
```

Comment:
I think it will be good to have a comment in the beginning of the `ceremony_inputs` fixture mentioning that key addition, removal and signing for `input_step_2` and `input_step4` are mocked using `ceremony_selection` fixture.